spec/lib/logicgraph_parser_spec.rb

```diff
@@ -20,7 +20,7 @@ def open_fixture(name)
 
     context 'parsing invalid XML' do
       it 'not throw an exception' do
-        expect(OntologyParser.parse open_fixture('broken.xml'), {}).
+        expect { OntologyParser.parse(open_fixture('broken.xml'), {}) }.
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.